src/app/services/game-play.service.spec.ts

```diff
@@ -0,0 +1,110 @@
+import { TestBed } from '@angular/core/testing';
+import { HttpClientTestingModule } from '@angular/common/http/testing';
+
+import { GamePlayService } from './game-play.service';
+import { AnswerService } from './answer.service';
+
+describe('GamePlayService', () => {
+  let service: GamePlayService;
+  let guessResult: string;
+  let acceptedGuessesResult: string[];
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      imports: [
+        HttpClientTestingModule
+      ],
+      providers: [AnswerService]
+    });
+    service = TestBed.inject(GamePlayService);
+    guessResult = '';
+    acceptedGuessesResult = [];
+  });
+
+  afterEach(() => {
+    jest.clearAllMocks();
+  })
+
+  it('should be created', () => {
+    expect(service).toBeTruthy();
+  });
+
+  it('should return A when A is added to currentGuess', () => {
+    let newLetter = 'A';
+
+    service.addLetter(newLetter);
+    service.currentGuess$.subscribe(currentGuess => guessResult = currentGuess);
+
+    expect(guessResult.includes(newLetter)).toBeTruthy()
+  });
+
+  it('should return ABCDE when A, B, C, D and E are added to currentGuess', () => {
+    let letters = 'ABCDE';
+
+    for (let i = 0; i < 5; i++) service.addLetter(letters[i]);
+    service.currentGuess$.subscribe(currentGuess => guessResult = currentGuess);
+
+    expect(guessResult.includes(letters)).toBeTruthy()
+  });
+
+  it('should return DORIT when DORITE is attempted for currentGuess', () => {
+    let letters = 'DORITE';
+
+    for (let i = 0; i < 6; i++) service.addLetter(letters[i]);
+    service.currentGuess$.subscribe(currentGuess => guessResult = currentGuess);
+    let canDoriteBeGuessed = 
+
+    expect(guessResult.includes('DORIT')).toBeTruthy()
+  });
+
+  it('should return LOL when removeLetter is called 3 times to currentGuess', () => {
+    let letters = 'LOLLY';
+
+    for (let i = 0; i < 5; i++) service.addLetter(letters[i]);
+    for (let i = 0; i < 2; i++) service.removeLetter();
+    service.currentGuess$.subscribe(currentGuess => guessResult = currentGuess);
+
+    expect(guessResult.includes('LOL')).toBeTruthy()
```

Comment:
A better test would be asserting that `guessResult === 'LOL'`, right? Since this test would still pass if `guessResult === 'LOLLY'`